.template/ios/Podfile

```
@@ -30,12 +30,66 @@ flutter_ios_podfile_setup
 target 'Runner' do
   use_frameworks!
   use_modular_headers!
-
+  
   flutter_install_all_ios_pods File.dirname(File.realpath(__FILE__))
 end
 
 post_install do |installer|
   installer.pods_project.targets.each do |target|
     flutter_additional_ios_build_settings(target)
+    target.build_configurations.each do |config|
+      # TODO You can enable the permissions needed here.
+      # permission, just remove the `#` character in front so it looks like this:
+      #
+      # ## dart: PermissionGroup.camera
+      #'PERMISSION_CAMERA=1'
+      #
+      #  Preprocessor definitions can be found in: https://github.com/Baseflow/flutter-permission-handler/blob/master/permission_handler/ios/Classes/PermissionHandlerEnums.h
+      config.build_settings['GCC_PREPROCESSOR_DEFINITIONS'] ||= [
+      '$(inherited)',
+      
+      ## dart: PermissionGroup.calendar
+      # 'PERMISSION_EVENTS=1',
+      
+      ## dart: PermissionGroup.reminders
+      # 'PERMISSION_REMINDERS=1',
+      
+      ## dart: PermissionGroup.contacts
+      # 'PERMISSION_CONTACTS=1',
+      
+      ## dart: PermissionGroup.camera
+      'PERMISSION_CAMERA=1',
+      
+      ## dart: PermissionGroup.microphone
+      # 'PERMISSION_MICROPHONE=1',
+      
+      ## dart: PermissionGroup.speech
+      # 'PERMISSION_SPEECH_RECOGNIZER=1',
+      
+      ## dart: PermissionGroup.photos
+      'PERMISSION_PHOTOS=1',
+      
+      ## dart: [PermissionGroup.location, PermissionGroup.locationAlways, PermissionGroup.locationWhenInUse]
+      'PERMISSION_LOCATION=1',
```

@doannimble Could you please turn off these permissions at the template level?

```suggestion
      ## dart: PermissionGroup.camera
      # 'PERMISSION_CAMERA=1',
      
      ## dart: PermissionGroup.microphone
      # 'PERMISSION_MICROPHONE=1',
      
      ## dart: PermissionGroup.speech
      # 'PERMISSION_SPEECH_RECOGNIZER=1',
      
      ## dart: PermissionGroup.photos
      # 'PERMISSION_PHOTOS=1',
      
      ## dart: [PermissionGroup.location, PermissionGroup.locationAlways, PermissionGroup.locationWhenInUse]
      # 'PERMISSION_LOCATION=1',
```

---

@luongvo Updated 🙏